src/components/autocomplete/js/autocompleteController.js

```diff
@@ -639,7 +679,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue ($event) {
+  function clearValue (shouldClearText) {
```

Comment:
We should remove any parameters here, because those weren't used before. 

Actually this method should just clear the autocompletes value and not validate anything.